docs/configuration.asciidoc

```diff
@@ -262,6 +262,40 @@ If set to `true`, the agent makes periodic requests to the APM Server to fetch t
 | true    | Boolean 
 |============
 
+[float]
+[[config-sanitize-field-names]]
+==== `SanitizeFieldNames` (added[1.2])
+
+Sometimes it is necessary to sanitize the data sent to Elastic APM, e.g. remove sensitive data.
```

Comment:
e.g. --> for example (**egg** sample)
i.e. --> in other words (**i**n **e**ssence)

Is removing sensitive data an example of sanitizing data? Or is it the same thing? If it's the same thing, which I think it is, this should be an i.e. instead.
```suggestion
Sometimes it is necessary to sanitize, i.e., remove, sensitive data sent to Elastic APM.
```

Reply:
Yeah, agreed, it's the same.